src/pages/settings/Security/TwoFactorAuth/CodesPage.js

```diff
@@ -0,0 +1,132 @@
+import React, {Component} from 'react';
+import {withOnyx} from 'react-native-onyx';
+import {View} from 'react-native';
+import _ from 'underscore';
+import PropTypes from 'prop-types';
+import HeaderWithCloseButton from '../../../../components/HeaderWithCloseButton';
+import Navigation from '../../../../libs/Navigation/Navigation';
+import ScreenWrapper from '../../../../components/ScreenWrapper';
+import withLocalize, {withLocalizePropTypes} from '../../../../components/withLocalize';
+import compose from '../../../../libs/compose';
+import ROUTES from '../../../../ROUTES';
+import FullPageOfflineBlockingView from '../../../../components/BlockingViews/FullPageOfflineBlockingView';
+import * as Illustrations from '../../../../components/Icon/Illustrations';
+import styles from '../../../../styles/styles';
+import FixedFooter from '../../../../components/FixedFooter';
+import Button from '../../../../components/Button';
+import Text from '../../../../components/Text';
+import Section from '../../../../components/Section';
+import TextFileLink from '../../../../components/TextFileLink';
+import ONYXKEYS from '../../../../ONYXKEYS';
+import Clipboard from '../../../../libs/Clipboard';
+
+const propTypes = {
+    account: PropTypes.shape({
+        /** User recovery codes for setting up 2-FA */
+        recoveryCodes: PropTypes.string,
+    }),
+    ...withLocalizePropTypes,
```

Comment:
always `withLocalizePropTypes` above the `account` to ensure nothing is overwritten